specification/ai/data-plane/HealthInsights/stable/2024-04-01/openapi.json

```diff
@@ -2108,10 +2127,23 @@
         "error": {
           "$ref": "#/definitions/Azure.Core.Foundations.Error",
           "description": "The error object."
+        },
+        "requestId": {
+          "type": "object",
+          "description": "An opaque, globally-unique, server-generated string identifier for the request.",
+          "properties": {
+            "response": {
+              "$ref": "#/definitions/Azure.Core.RequestIdResponseHeader"
+            }
+          },
+          "required": [
+            "response"
+          ]
         }
       },
       "required": [
-        "error"
+        "error",
+        "requestId"
```

Comment:
I don't see where these changes comes from.

There is no update to typespec file about these in this PR?

Reply:
@weidongxu-microsoft 
Maybe caused by new compiler version?